Content.Server/GameObjects/Components/GUI/InventoryComponent.cs

```diff
@@ -392,5 +392,29 @@ public override ComponentState GetComponentState()
             }
             return new InventoryComponentState(list);
         }
+
+        void IExAct.OnExplosion(ExplosionEventArgs eventArgs)
+        {
+            switch (eventArgs.Severity)
+            {
+                case ExplosionSeverity.Destruction:
+                case ExplosionSeverity.Heavy:
+                    break;
+                default:
+                    return;
+            }
```

Comment:
Why not just return if the explosion severity is light here instead of using a switch?

Reply:
In case more severities are added, since destruction is value 0. Should I check for Light specifically or for a severity value under 2?